src/main/java/bh/bot/Main.java

```diff
@@ -79,7 +74,8 @@ public static void main(String[] args) {
 					TrialsApp.class, //
 					GvgApp.class, //
 					GauntletApp.class, //
-					GenMiniClient.class,
+					GenMiniClient.class, //
+					ChangeCharacterApp.class,
```

Comment:
@daredoes you forgot this to make app works

Reply:
Btw how do you think if we change the app code from `character` to `change-character` to make it more meaning full?

Reply:
Oops! I refactored this app out from a branch where I had both apps being developed, and forgot this line!

Totally open to changing the character code. I've tried not to spend too much time getting lost in the nitty gritty of making the feature work, as the goal is to play the game not have it play me 😄

Reply:
Changed app code into `change-character`